<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityDialog</class>
 <widget class="QDialog" name="ActivityDialog">
  <property name="geometry">
   <rect>
       <x>0</x>
       <y>0</y>
       <width>340</width>
       <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activity</string>
  </property>
     <layout class="QVBoxLayout" name="verticalLayout">
         <item>
             <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                     <widget class="QLabel" name="lbName">
                         <property name="text">
                             <string>Name</string>
                         </property>
                     </widget>
                 </item>
                 <item row="0" column="1">
                     <widget class="QLineEdit" name="leName"/>
                 </item>
                 <item row="1" column="0">
                     <widget class="QLabel" name="lbCustomer">
                         <property name="text">
                             <string>Customer</string>
                         </property>
                     </widget>
                 </item>
                 <item row="1" column="1">
                     <widget class="QComboBox" name="cbCustomer"/>
                 </item>
                 <item row="2" column="0">
                     <widget class="QLabel" name="lbProject">
                         <property name="text">
                             <string>Project</string>
                         </property>
                     </widget>
                 </item>
                 <item row="2" column="1">
                     <widget class="QComboBox" name="cbProject"/>
                 </item>
             </layout>
         </item>
         <item>
             <spacer name="verticalSpacer">
                 <property name="orientation">
                     <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                     <size>
                         <width>20</width>
                         <height>1</height>
                     </size>
                 </property>
             </spacer>
         </item>
         <item>
             <widget class="QDialogButtonBox" name="buttonBox">
                 <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="standardButtons">
                     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
                 </property>
             </widget>
         </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActivityDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActivityDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
